topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorTest.java

```diff
@@ -608,12 +612,12 @@ public void testReconcile_noResource_withKafka_noPrivate(VertxTestContext contex
         Topic kafkaTopic = new Topic.Builder(topicName.toString(), 10, (short) 2, map("cleanup.policy", "bar"), metadata).build();
         Topic privateTopic = null;
 
-        CountDownLatch async0 = new CountDownLatch(1);
+        CountDownLatch topicCreatedInKafka = new CountDownLatch(1);
```

Comment:
@tombentley  so I went a little over the top renaming everything called `async0`, hopefully captured what was being expected correctly.